LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019-2021 Hao Hou <haohou302@gmail.com>
+Copyright (c) 2019-2025 Hao Hou <haohou302@gmail.com>
```

Comment:
That's a good idea, but I'm not sure whether it should be "plotters maintainers" instead of Hao Hou since they haven't been active since 2023.

```suggestion
Copyright (c) 2019-2022 Hao Hou <haohou302@gmail.com>
Copyright (c) 2022-2025 The plotters-rs contributors
```